src/app.ts

```diff
@@ -7,7 +7,7 @@ const debug = Debug('universalmediaserver-api:server');
 
 import indexRouter from './routes/index';
 import mediaRouter  from './routes/media';
-import { ExternalAPIError, MediaNotFoundError } from './helpers/customErrors';
+import { ExternalAPIError, MediaNotFoundError, IMDbIDNotFoundError } from './helpers/customErrors';
```

Comment:
I think we either stick with the generic `MediaNotFoundError` or, add both an imdb one as well as an open subs one

Reply:
The error means we couldn't identify what the IMDb ID was based on the title search, so it's not specific to one external API